templates/monitoring/datadog.tf.tpl

```diff
@@ -88,8 +93,8 @@ resource "datadog_monitor" "pod_restarts" {
 
 resource "datadog_monitor" "pod_restarts_high" {
   type = "query alert"
-  name = "{{ .Config.Name | title }} Pod Restarts > ${var.pod_restart_threshold} last 30m"
```

Comment:
This is a breaking change, right?

Reply:
new variable defined in the same file, **pod_restart_high_count_threshold**. would it be a breaking change?

Reply:
confirmed that pod_restart_threshold var is not overridden in terraform.tfvars

https://github.com/search?q=org%3Agetoutreach+pod_restart_threshold+filename%3Aterraform.tfvars&type=Repositories&ref=advsearch&l=&l=

![Screenshot 2023-03-01 at 10 52 53 AM](https://user-images.githubusercontent.com/93616024/222237600-0e75cf3d-b328-4309-9128-b15f5f4d1d66.png)